ui/src/app/utils/stores.tsx

```diff
@@ -255,6 +256,13 @@ export function withParamsKey(...paramNames: (keyof MatchParams)[]) {
     };
 }
 
+export interface UserAppsStore {
+  updating?: boolean;
```

Comment:
should these be non-optional?  we should always have these values if the store is populated, right?